CefSharp.WinForms/ChromiumWebBrowser.cs

```diff
@@ -273,25 +288,76 @@ private static void OnApplicationExit(object sender, EventArgs e)
         /// <summary>
         /// Initializes a new instance of the <see cref="ChromiumWebBrowser"/> class.
         /// </summary>
+        public ChromiumWebBrowser()
+        {
+
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ChromiumWebBrowser"/> class.
+        /// </summary>
         /// <param name="address">The address.</param>
-        /// <exception cref="System.InvalidOperationException">Cef::Initialize() failed</exception>
         public ChromiumWebBrowser(string address)
         {
-            if (!Cef.IsInitialized && !Cef.Initialize())
+            Address = address;
+        }
+
+        private void Initialize()
```

Comment:
More descriptive naming please. I'm wondering if this method should only contain the `unmanaged` calls.

Setting `Dock` seems to have vanished, need to double check nothing else has been missed.